### DISCUSSION TITLE
Is there a way to bind a single button to cast all spells in recommendation que?

### DISCUSSION THREAD

**Discussion by ryceworx** [Hekili/hekili#1075] [Q&A] [answered]
Hello.
I do quests to raise my twink and don't really want to think about sequences when completing same quest 999 times. So there is a question: is there an ability to use only one button to cast every spell from rotation Hekili recommends? For example: My spells binded to 1,2,3 keyboard buttons. Hekili rotation looks like: 3,2,1. I'd like to use only F1 every time to tell Hekili use next spell from rotation,  It's kinda single button gaming :D I think it would be very helpful in routine. 
Thanks for your help!

**Comment by Hekili** [comment, marked as answer]
No.

**Reply by ryceworx** [reply]
Ok. Thank you for a fast reply. Would it possible to add it in future?

**Reply by Hekili** [reply]
No.
